.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+@saradrada @garzuzo @holgiosalos @danielgalvis98 @leonleo997 @manuelq12
```

Comment:
```suggestion
* @saradrada @garzuzo @holgiosalos @danielgalvis98 @leonleo997 @manuelq12
```

Reply:
**Remember:** Each line is a file pattern followed by one or more owners. **The asterisk means every file.**

Reply:
Added new commit with edited code owners file